bitchat/Models/CommandsInfo.swift

```diff
@@ -0,0 +1,60 @@
+//
+// CommandsInfo.swift
+// bitchat
+//
+// This is free and unencumbered software released into the public domain.
+// For more information, see <https://unlicense.org>
+//
+
+import Foundation
+
+// MARK: - CommandInfo Enum
+
+enum CommandInfo: String, CaseIterable, Identifiable {
+    case block = "/block"
+    case clear = "/clear"
+    case hug = "/hug"
+    case message = "/m, /msg"
+    case slap = "/slap"
+    case unblock = "/unblock"
+    case who = "/w"
+    case favorite = "/fav"
+    case unfavorite = "/unfav"
+
+    var id: String { rawValue }
+
+    var localizedSyntax: String? {
+        switch self {
+        case .block, .hug, .message, .slap, .unblock, .favorite, .unfavorite:
+            return NSLocalizedString("content.input.nickname_placeholder", comment: "")
```

Comment:
looks like "syntax" in the old code sometimes is "<nickname>" and sometimes "[nickname]" - also, not sure if it's localized